Source/AnimationDescription.swift

```diff
@@ -11,11 +11,12 @@ import UIKit
 public final class AnimationDescription {
     let item: AnimatableItem
     let attribute: AnimationAttribute
-    let type: AnimationType
```

Comment:
Where did the type go?

Reply:
AnimationType is unnecessary now.
It was used to split usage of motion animations which were using `path` and basic animation, which were animating properties to any value.

Now all of animations are using `CAKeyframeAnimation`. Will remove this type in next commit